src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1526,6 +1523,13 @@ public String toVariableNameString(EnchantmentOffer eo) {
 				.name("Transform Reason")
 				.description("Represents a transform reason of an <a href='events.html#entity transform'>entity transform event</a>.")
 				.since("2.8.0"));
+
+		Classes.registerClass(new ClassInfo<>(Beacon.class, "beacon")
```

Comment:
this shouldn't exist, all the syntaxes should act on blocks and just check if the block is a beacon.
that should change this pr enough that i will save further reviews for after this is addressed.